query-service/src/main/resources/configs/common/application.conf

```diff
@@ -3,10 +3,12 @@ service.name = query-service
 service.port = 8090
 service.admin.port = 8091
 service.config = {
+  tenantColumnName = ${?TENANT_COLUMN_NAME}
```

Comment:
what is default value?

Reply:
There is no default for this, we want it to explicitly set.

Reply:
@ravisingal I think we can set the default value directly to `tenant_id` here instead of ENV variable.

Reply:
sounds good here as we have already hardcoded this in schema, we can do that.

Reply:
Done.